src/functions-templates/go/scheduled-function/go.mod

```diff
@@ -0,0 +1,5 @@
+module github.com/someone/{{name}}
```

Comment:
do you want to indicate that they'll likely have to modify this?

Reply:
This `{{name}}` placeholder is automatically replaced by the name configured for the template. I can add a comment if that would make it obvious.

Reply:
yeah i was mostly talking about the `someone` here